apps/spectator/package.json

```
@@ -8,6 +8,12 @@
     "start": "next start",
     "lint": "eslint . --max-warnings 0"
   },
+  "lint-staged": {
+    "**/*.{js,jsx,ts,tsx}": [
+      "prettier --write",
+      "eslint --fix"
```

```suggestion
      "eslint --fix --max-warnings 0"
```

최대 경고 수를 지정하여 에러를 발생시키는 코드가 linting pass 되지 않도록 하는 건 어떨까요? 다른 lint-staged에서도 모두 해당되는 옵션인 것 같습니다!

---

좋은 의견인 것 같네요 추가해서 올리도록 하겠습니다!